README.md

```diff
@@ -1,82 +1,163 @@
-# Math
+# Maths
 
+## Objectives
 
+1. Perform mathematical operations on integer values
+1. Describe the `Double` data type
+1. Learn how to create functions performing mathematical operations
+1. Introduce the full set of mathematical operators
 
-## Learning Objectives - The student should be able to..
+## Arithmetic operators
 
-* Set variables where the value's are of type `Int` and then perform math operations on those variables, like so:
+You have already learned about the type `Int` which can be used to create variables and constants holding single integer numbers:
 
 ```swift
-let combinedAgesOfFamily = 30 + 29 + 64 + 65
-// 188
-
 let myAge = 30
 let brothersAge = 29
 let momsAge = 64
 let dadsAge = 65
-let combinedAgesOfFamily2 = myAge + brothersAge + momsAge + dadsAge
-// 188
 ```
 
-* Create variables of type `Double`. 
-* Create a function like this on their own:
+Commonly, once we have numbers, we want to perform mathematical operations on them, for example summing them up. For doing this, Swift provides *operators*, special symbols that you use to check, combine or change one or more values. 
+
+The symbols used are similar to the ones you would use on paper, like `+` for performing additions:
 
```

Comment:
Your approach here is exactly how we like approaching topics (from beginner to advanced level). Making no assumptions and guiding them into something that's new to them referring back to something they already know how to do.